components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -47,6 +47,7 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 			}
 			:host([shown]) {
 				display: inline-block;
+				overflow: auto;
```

Comment:
I think the original purpose of the `overflow: hidden` was to keep the contents of the parent views hidden when drilled in. For example, if the parent view is taller than the nested view, it would result in a scrollbar. This can be seen by drilling into View 3 on the hierarchical-view demo page, which is configured to capture this case.

Showing the scrollbar for the active view might also result in some bumping while the height is being animated. I didn't observe that while looking at your change, but that would be a separate thing to check more closely.

![image](https://github.com/user-attachments/assets/5d59c5a0-4207-4881-be56-a16d0a981607)

Reply:
Sorry I should have clarified in the description - this is for the flag off case and not the flag on case. With the `left` change this would still be a problem for the flag off case.

Reply:
This seems to work as expected if instead I move the `overflow: auto` to be specifically on the `hierarchical-view` in filter ([here](https://github.com/BrightspaceUI/core/blob/main/components/filter/filter.js#L219)). Are there concerns that I'm missing with this approach?

Reply:
Could that have the same problem? Like if the inner view is shorter than the outer view?

Reply:
I tried it out and it seemed to be fine. I believe this is because the parent view no longer has the `overflow: auto`, just the child.

Reply:
Ok, so that would be temporary until we remove the fixed positioning flag, yeah?

Reply:
It would, yup. I'll add a comment with that